webrtc.html

```diff
@@ -9043,17 +9043,17 @@ <h2 id="sec.stats-model">Statistics Model</h2>
         bytesReceived</li>
         <li>RTCMediaStreamStats, with attributes streamIdentifer, trackIds</li>
         <li>RTCMediaStreamTrackStats, with attributes trackIdentifier,
-        remoteSource, ended, detached, ssrcIds, frameWidth, frameHeight,
+        remoteSource, ended, detached, frameWidth, frameHeight,
         framesPerSecond, framesSent, framesReceived, framesDecoded,
         framesDropped, framesCorrupted, audioLevel</li>
         <li>RTCCodecStats, with attributes payloadType, codec, clockRate,
-        channels, parameters</li>
```

Comment:
Sounds right. Though I am not familiar enough to be sure that `parameters` and `sdpFmtpLine` are related.

I have added `sdpFmtpLine` and others can help verify if it is acceptable.